.eslintrc.yaml

```diff
@@ -49,12 +49,12 @@ rules:
 settings:
   react:
     version: 'detect'
-  import/ignore:
-    - /test-utils/
-    - test-utils$
-    - test-utils\.js$
+  # import/ignore:
+  #   - /test-utils/
+  #   - test-utils$
+  #   - test-utils\.js$
```

Comment:
Hm?

Reply:
Yeah, not sure why but eslint was complaining about `import/named`

<img width="849" alt="image" src="https://user-images.githubusercontent.com/1110551/73305761-14e00f80-421b-11ea-81e2-a22b005a56b8.png">

CI was also failing

<img width="1326" alt="image" src="https://user-images.githubusercontent.com/1110551/73305805-2e815700-421b-11ea-8edb-eafbd1ec1221.png">

I can't remember exactly why we had those rules...

Reply:
Strange. Would we get those in the MC too? Does intellisense work there?

Reply:
I suppose because eslint doesn't recognize the ts files from the test-utils import, which are being "imported" in a js file. Those tests are in the playground app, I suppose because the package does not have any tsconfig.

Reply:
> Would we get those in the MC too? Does intellisense work there?

Once bundled, there should be no problem.

Reply:
<img width="267" alt="image" src="https://user-images.githubusercontent.com/1110551/73306229-1231ea00-421c-11ea-81fa-2d5dcffef34a.png">

Reply:
Should we remove the comment then before merging?

Reply:
I'll keep it there for now, just in case we bump into that issue again